core/src/main/java/org/elasticsearch/search/aggregations/bucket/children/ParentToChildrenAggregator.java

```diff
@@ -225,8 +225,9 @@ public ChildrenAggregatorBuilder(String name, String childType) {
             if (childDocMapper != null) {
                 ParentFieldMapper parentFieldMapper = childDocMapper.parentFieldMapper();
                 if (!parentFieldMapper.active()) {
+                    // NOCOMMIT fix exception args
                     throw new SearchParseException(aggregationContext.searchContext(),
```

Comment:
Any suggestions as to what this exception should be now? It's not really to do with parsing anymore. I was thinking IllegalStateException but maybe that's too generic?